test/lib/mayaUsd/render/mayaToHydra/CMakeLists.txt

```diff
@@ -36,7 +36,7 @@ set(INTERACTIVE_TEST_SCRIPT_FILES
     testOpenPBRSurface.py
     testFlowViewportAPI.py
     testStageVariants.py|skipOnPlatform:osx # HYDRA-1127 : refinedWire not working on OSX
-    testStagePayloadsReferences.py
+    #testStagePayloadsReferences.py #Disabled for usd 24.11
```

Comment:
Could you put the re-enabling jira id here?

Reply:
Sure